### PR TITLE
Add scrapple as a comestible

### PR DESCRIPTION
**Pull request by UrbanKeel**
#### Summary
Features "Add scrapple as a comestible"

#### Purpose of change

Scrapple's a pretty common food around New England, thought it'd make sense to be in the game. Also it's a very simple meat-and-grain mush, and CDDA has surprisingly few meat-and-grain sausages/mushes despite them being a common peasant food.

#### Describe the solution
Add scrapple as a food item in game, as well as its recipe.

#### Describe alternatives you've considered
Not adding scrapple

#### Testing
Tested the recipe in-game, seems to work fine. Also made sure all the different types of the recipe made sense in real life and in-game. Also-also checked to make sure its sprite looked like cooked offal.

### CODE REVIEW

**Review by LyleSY — APPROVED** [review]
This all makes sense to me

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -2001,5 +2001,26 @@
...
     "vitamins": [ [ "calcium", "423 mg" ], [ "iron", "8 mg" ] ]
+  },
+  {
+    "id": "scrapple",
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "name": { "str_sp": "scrapple" },
+    "symbol": "%",
+    "looks_like": "offal_cooked",
+    "description": "A dry mush of meat scraps and grain, sliced and pan-fried; better than it sounds and looks. A staple food of the Mid-Atlantic United States.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/meat_dishes.json:2012]:
```suggestion
    "description": "A dry mush of meat scraps and grain, sliced and pan-fried; better than it sounds and looks.  A staple food of the Mid-Atlantic United States.",
```

**Comment by Maleclypse** [comment]
Thanks for your contribution!

**Comment by UrbanKeel** [comment]
Discord name Antinkler btw

**Pull request merged**